lib/utils/src/graph/algorithms.cc

```diff
@@ -327,6 +380,14 @@ std::vector<DirectedEdge> get_edge_topological_ordering(DiGraphView const &g) {
   return result;
 }
 
+std::vector<Node> get_neighbors(DiGraphView const &g, Node const &n) {
```

Comment:
Return type here should be an `unordered_set`, not a `vector`

Reply:
Done